tests/utils/test_xml_helpers.py

```
@@ -0,0 +1,86 @@
+            ],
+            "Science": [
+                Feed(
+                    title="ScienceDaily",
+                    url="http://sciencedaily.com/feed",
+                    genre="Science",
+                    description="Science news",
+                )
+            ],
+        }
+
+    def test_create_opml_tree_valid_feed_data(self):
```

Ruff (0.8.0)

Missing return type annotation for public function `test_create_opml_tree_valid_feed_data`

Add return type annotation: `None`

(ANN201)